install/turn-things-off.sh

```diff
@@ -5,9 +5,6 @@ if [[ -n "$MINIMIZE_DOWNTIME" ]]; then
   $dc rm -fsv $($dc config --services | grep -v -E '^(nginx|relay)$')
 else
   # Clean up old stuff and ensure nothing is working while we install/update
-  # This is for older versions of on-premise:
-  $dc -p onpremise down -t $STOP_TIMEOUT --rmi local --remove-orphans
-  # This is for newer versions
```

Comment:
The older/newer distinction is from https://github.com/getsentry/self-hosted/pull/316 which is almost 2 years old. I think we can do away with the older version by now.

Reply:
Yes as people need to go through some other hard-stops anyway.